core_backend/app/llm_call/utils.py

```diff
@@ -1,11 +1,35 @@
-from litellm import completion
+from litellm import acompletion, completion
 
 from ..configs.app_config import LLM_ENDPOINT, LLM_MODEL
 from ..utils import setup_logger
 
 logger = setup_logger("LLM_call")
 
 
+async def _ask_llm_async(question: str, prompt: str) -> str:
```

Comment:
We should switch to this async version for other calls and deprecate the other one. I will do that in a different PR

Reply:
Sounds good - could you add a ticket for this?

Reply:
Sure! I've created it and will also address it in a PR today / tomorrow.

https://idinsight.atlassian.net/browse/AAQ-287